src/components/Navbar.tsx

```diff
@@ -32,7 +31,7 @@ const Navbar = () => {
             {
               label: t('actions.new'),
               onClick: () => {
-                history.push('/patients/new')
+                history.push('/new/patients')
```

Comment:
Why changing this line? I think it is better to got with `/entityName(s)/new` for every entity we have

Reply:
I'm considered that if we have route such as 

`/entity/:id`
and 
`/entity/new`

That we could end up with collisions. 

This guarantees that we will never have that happen.

Reply:
I updated the route back to what it was. 😊